core/domain/user_jobs_one_off.py

```diff
@@ -407,6 +407,35 @@ def reduce(key, values):
         yield (key, len(values))
 
 
+class AddTopicIdsAndStoryIdsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that adds story_ids and topic_ids in CompletedActivitiesModel.
+
+    NOTE TO DEVELOPERS: This job can be deleted after it is run in April 2021
+    release.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [
+            user_models.CompletedActivitiesModel,
+            user_models.IncompleteActivitiesModel
+        ]
+
+    @staticmethod
+    def map(model_instance):
+        model_instance.story_ids = []
+        model_instance.topic_ids = []
```

Comment:
This is not needed, repeated properties are `[]` by default.

Reply:
Done. PTAL!